tile_generator/test_config_expected_output.json

```diff
@@ -4256,6 +4256,13 @@
     }
   ],
   "sha1": false,
+  "git_remotes": [
+    {
+      "name":  "origin",
+      "urls": ["git@example.com:org/repo.git"]
+    }
+  ],
+  "git_sha": "some-sha",
```

Comment:
The template above in the metadata.yml file Line 57 has git_sha before git_remotes. Is this out of order here?

Reply:
Thanks, I'll check this.